app/main.py

```diff
@@ -1 +1,62 @@
-# write your code here
+from typing import List
+
+
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name: str = name
+        self.skills: List[str] = []
+
+    def learn_skill(self, skill: str) -> None:
+        if skill not in self.skills:
+            self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        default_skills = ["JavaScript", "HTML", "CSS"]
+        for skill in default_skills:
+            self.learn_skill(skill)
```

Comment:
You don't have to add each skill through the `learn_skill()` method because there is an empty(!) `skills` attribute.
You can just add a whole list of default skills to `skills` attribute.

Same with `BackendDeveloper` and `AndroidDeveloper` classes.